popSimManu.tex

```diff
@@ -474,13 +474,80 @@ \subsection*{Model QC procedures}
+to exhaustively test the performance of these methods, but rather to explore their behavior
+in two different inference scenarios and demonstrate the potential for this resource to
+eventually be used for more in-depth benchmarking. As a result, these methods were used
+generally with default settings and we did not attempt to optimize their performance or fit
+parameter-rich demographic models. As with the single population inference pipeline,
+we developed a \texttt{snakemake} (\cite{koster2012snakemake}) workflow to manage
+our inference pipeline and maintain reproducibility.
+
+For both \dadi and \fastsimcoal2, our general approach was to fit a two population
+isolation with migration (IM) model with constant population sizes to the simulated
+data from the human and \emph{Drosophila} models. This IM model contains six parameters:
+the ancestral population size, the size of population 1 after split, the size of
+population 2 after split, the divergence time, and two migration rate parameters. Importantly,
+this meant that for both the human and \emph{Drosophila} inference, the fitted model
+did not match the simulated model (Figure \ref{fig:IM_popn_human}C), which, for humans,
+includes three populations with various population size changes as well as exponential
+growth. In the human case, we therefore performed inference solely on the Africa (YRI)
+and Europe (CEU) populations, meaning that the Asia (CHB) population functioned as a ghost
+population that was ignored by our inference. Our motivation for this simple
+approach was to mimic the typical approach of two population inference on empirical
+data, where the user is not aware of the 'true' underlying demography and the inference
+model is often misspecified. To ground-truth our inference approach, we also conducted
+inference on a generic IM model that was identical to the model used for inference.
+% CK: I have a figure of the generic IM results in the works that we can include in the supplement
+
+To perform inference, we began by running simulations under each model, performing
+three replicates to inspect for inter-replicate variability. For humans, we ran
+whole genome simulations (chromosomes 1-22) under the Gutenkunst model (\cite{gutenkunst2009inferring}),
+using the HapMap II recombination map (\cite{international2007second}) and taking 20
+samples each from the CEU and YRI populations. For \emph{Drosophila},
+we simulated 3 Mb of chromosome 2R under the Li & Stephan model (\cite{li2006inferring}),
+using the map inferred by \cite{comeron2012many} and taking 50 samples from each of
+the Africa and Europe populations. Due to computational limitations, we were unable
+to run whole genome simulations for \emph{Drosophila}. For the generic IM simulations,
+we used the human genome (chromosomes 1-22) and HapMap II recombination map (\cite{international2007second})
+and took 20 samples per population.
+% CK: should we say more about the drosophila simulations here?
```

Comment:
maybe just a sentence that explains the out of africa nature of the drosophila model?

Reply:
ok can do. should we say anything more about our inability to do whole genome drosophila sims for that model?

Reply:
yes, but let's get @jradrion to chime in with the details once we merge your PR. maybe you can just leave a stub in a sentence like "We tried to run whole genome simulations under this model but after XXXX amount of time..."